cpp/CMakeLists.txt

```diff
@@ -229,6 +236,9 @@ if(NOT BUILD_SHARED_LIBS)
   endif()
 endif()
 
+# need to add nanoarrow to the export set
+install(TARGETS nanoarrow EXPORT cudf-exports)
+
```

Comment:
This should be superfluous now with the use of `rapids_cpm_find`

Reply:
Turns out it's not, at least I couldn't figure out how to make it unnecessary. I still get an error about `nanoarrow` not being in the export set unless I have this line. If you have any suggestions about how I can modify the `rapids_cpm_find` invocation to fix that, I'll give it a try.